pennylane/gradients/gradient_transform.py

```diff
@@ -278,18 +278,22 @@ def default_qnode_wrapper(self, qnode, targs, tkwargs):
         # to take into account that classical processing may be present
         # inside the QNode.
         hybrid = tkwargs.pop("hybrid", self.hybrid)
-
-        old_interface = qnode.interface

```

Comment:
why are you removing the old_interface stuff in this PR?

Reply:
This was introduced by me and it was wrong to do it there because those are the transform arg not the qnode arg...